Extension/lib/prefs.js

```diff
@@ -48,7 +48,11 @@ var Prefs = (function () {
             }
             return browser;
         })(),
-        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0
+        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0,
```

Comment:
Hm, this is something specific to stealth mode i guess. We just should not migrate it to the main extension.